src/tactic/explode_widget.lean

```diff
@@ -0,0 +1,305 @@
+/-
+Copyright (c) 2020 Minchao Wu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Minchao Wu
+-/
+import tactic.explode
+import tactic.interactive_expr
+/-!
+# `#explode_widget` command
+
+Render a widget that displays an `#explode` proof, providing more 
+interactivity such as jumping to definitions and exploding constants 
+occurring in a proof term subsequently.
```

Comment:
Can you reword this? I'm not sure what "exploding constants occurring in a proof term subsequently" means.